src/stirling/bpf_tools/task_struct_resolver.cc

```diff
@@ -137,8 +137,54 @@ StatusOr<TaskStructOffsets> ScanBufferForFields(const struct buf& buf,
 
   TaskStructOffsetsManager offsets_manager(&task_struct_offsets);
 
+#if AARCH64
+  // On aarch64, the cpu_context struct inside the thread_struct, can contain the same address as
```

Comment:
I'm not sure if there is a good reference to include, but if so, maybe a pointer to that. It would be neat to have a link that pulls up the `thread_struct` source code.